pkg/instancetype/instancetype.go

```diff
@@ -95,6 +103,14 @@ func (m *methods) createInstancetypeRevision(vm *virtv1.VirtualMachine) (*appsv1
 		if err != nil {
 			return nil, err
 		}
+
+		// There is still a window where the instancetype can be updated between the VirtualMachine validation webhook accepting
+		// the VirtualMachine and the VirtualMachine controller creating a ControllerRevison. As such we need to check one final
+		// time that there are no conflicts when applying the instancetype to the VirtualMachine before continuing.
+		if err = m.checkForInstancetypeConflicts(&instancetype.Spec, vm.Spec.Template.Spec.DeepCopy()); err != nil {
```

Comment:
Can you move the `DeepCopy()` call to the inside of the function `checkForInstancetypeConflicts()`? I would expect a function called `check*` to not modify its parameters or have side effects.

Reply:
ACK done, thanks!